src/main/java/core/basesyntax/service/operation/SupplyOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+public class SupplyOperation implements OperationHandler {
+    @Override
+    public int getQuantity(int defaultQuantity, int fruitQuantity) {
```

Comment:
The method name `getQuantity` is not very descriptive of the operation performed. Consider renaming it to more accurately reflect that it's handling a 'supply' operation, such as `applySupplyOperation` or `calculateSupplyTotal`.